src/django_oapif/decorators.py

```diff
@@ -1,25 +1,28 @@
+import json
 from typing import Any, Callable, Dict, Optional
 
+from django.contrib.gis.geos import GEOSGeometry
 from django.db import models
 from django.db.models.functions import Cast
 from rest_framework import serializers, viewsets
-from rest_framework.serializers import ModelSerializer
 from rest_framework_gis.serializers import GeoFeatureModelSerializer
 
 from django_oapif.metadata import OAPIFMetadata
 from django_oapif.mixins import OAPIFDescribeModelViewSetMixin
 from django_oapif.urls import oapif_router
 
 from .filters import BboxFilterBackend
-from .functions import AsGeoJSON
 
-USE_PG_GEOJSON = False
-USE_PG_GEOJSON = True
+# this should be be switched to contrib
+# when https://code.djangoproject.com/ticket/34882#ticket is fixed
+from .functions import AsGeoJSON
 
 
 def register_oapif_viewset(
     key: Optional[str] = None,
-    skip_geom: Optional[bool] = False,
+    geom_db_serializer: Optional[bool] = True,
```

Comment:
The naming is a bit confusing, I thought you are passing a `drf.Serializer` instance here. Wouldn't it make more sense to move it in `settings.py` as a global config? Is there a situation to use both in the same time? I though we have this branching in logic only for testing and in the future only one will survive?

Reply:
I was thinking of a geometry returned by a getter instead of a db field in which scenario you cannot pass it. At the moment it would makes it also a bit more powerful/easy for testing/profiling.

what about `serialize_geom_in_db` ?